tests/core/test_attack_matrix.py

```
@@ -0,0 +1,285 @@
+        provider = FallbackProvider()
+        tactics = provider.get_tactics()
+        assert "credential-access" in tactics
+        assert "initial-access" in tactics
+        assert "lateral-movement" in tactics
+
+    def test_tactic_info_structure(self):
+        from athf.core.attack_matrix import FallbackProvider
+
+        provider = FallbackProvider()
+        tactics = provider.get_tactics()
+        for key, info in tactics.items():
```

Ruff (0.15.6)

[warning] Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)